pkg/csi/cinder/driver.go

```diff
@@ -68,6 +68,8 @@ type Driver struct {
 	endpoint  string
 	clusterID string
 
+	withTopology bool
```

Comment:
can we group it with the endpoint and clusterID members?

Reply:
Done